src/spec-node/collectionCommonUtils/generateDocs.ts

```diff
@@ -0,0 +1,38 @@
+import { Argv } from 'yargs';
+import { CLIHost } from '../../spec-common/cliHost';
+import { Log } from '../../spec-utils/log';
+
+const targetPositionalDescription = (collectionType: GenerateDocsCollectionType) => `
+Generate docs of ${collectionType}s at provided [target] (default is cwd), where [target] is either:
+   1. A path to the src folder of the collection with [1..n] ${collectionType}s.
+   2. A path to a single ${collectionType} that contains a devcontainer-${collectionType}.json.
+`;
+
+export function GenerateDocsOptions(y: Argv, collectionType: GenerateDocsCollectionType) {
+	return y
+		.options({
```

Comment:
Should we keep `--project-folder` and `-p` as a hidden option with a warning message in the log when used for compatibility? Since this has been available for a few weeks users might have picked it up and removing it could break things locally or in CIs.